ecosystem/sep-0024.md

```diff
@@ -437,9 +438,10 @@ After the user has provided all necessary information to the anchor through the
 
 When the popup is closed, the transaction's status should no longer be `incomplete`. Instead, it should be one of the following:
 
-* `pending_user_transfer_start` if anchor is ready to receive the funds. This is the most common status after `incomplete`.
 * `pending_anchor` if the anchor must verify KYC information given prior to receiving funds.
 * `pending_user` if the user must take some action before sending funds to the anchor.
+* `pending_user_transfer_start` if anchor is ready to receive the funds. This is the most common status after `incomplete`.
+* `pending_user_transfer_complete` if anchor detects the payment is fulfilled and the funds is ready for the user to pick up. This is the most common status after `pending_user_transfer_start` the user must pick up the funds in person.
```

Comment:
I wouldn't say this status value is the most common after `pending_user_transfer_start` because this is only true with withdrawal transactions and transactions using cash.